lib/features/app_changelog/widgets/app_changelog_list.dart

```diff
@@ -0,0 +1,72 @@
+import "package:flutter/material.dart";
+
+import "../../../theme/app_theme.dart";
+import "../repository/changelog_repository.dart";
+
+class AppChangelogList extends StatelessWidget {
+  const AppChangelogList({super.key, required this.changelog});
+
+  final Changelog changelog;
+
+  @override
+  Widget build(BuildContext context) {
+    return Flexible(
+      child: ListView.separated(
+        shrinkWrap: true,
+        itemBuilder: (BuildContext context, int index) => _ListItem(
+          tag: changelog.changes![index]!.changeType,
```

Comment:
uhh - I lot of `!` signs. I do not like it - we should avoid force unwrapping in all cases possible, just to be safe

Reply:
you can think of some default values for some fields or hide the whole ui list item if there's a null in some important place. do as you please, but something more safe please